exo-setup/src/app-setup.ls

```diff
@@ -37,7 +37,7 @@ class AppSetup extends EventEmitter
       async.map-series
     else
       async.map
-    operation setups, (-> &0.start &1), (err) ~>
+    async.map-series setups, (-> &0.start &1), (err) ~>
```

Comment:
The `docker-setups` can still be done in parallel on non-Windows systems, so should we still use `async.map` for those, or just do it all synchronously here?